mzLib/Proteomics/Fragmentation/DissociationTypeCollection.cs

```diff
@@ -102,6 +102,25 @@ public static List<ProductType> GetWaterAndAmmoniaLossProductTypesFromDissociati
             { ProductType.D, null},// diagnostic ions are not shifted but added sumarily
             { ProductType.Ycore, null},// neutral Molecular product can be used with neutral loss as fragment
             { ProductType.Y, null},// diagnostic ions are not shifted but added sumarily
+
+            // Rna Specific types are not shifted through this method, but added to not break the code. 
```

Comment:
why change the standard protocol?

Reply:
Because both rna and proteins have a a,b,c,x,y,z ions which differ in their neutral losses. One central location for neutral losses would not be possible without specifying which type of bio-polymer you are interested in. Implementing the method in this fashion would require all calls in MetaMorpheus to be altered

Reply:
Comments were added